docs/config.md

```diff
@@ -164,6 +164,8 @@ An HTML fragment to add to the header. Defaults to the empty string.
 
 An HTML fragment to add to the footer. Defaults to “Built with Observable.”
 
+head, header and footer can be specified as strings, or as functions that receive as arguments the page’s title, front matter, and path, and return a string.
```

Comment:
I think this needs to be repeated for each option. Otherwise you’re likely to miss it when you’re looking at **head** or **header** — you won’t know that this sentence applies to multiple options. It would also be nice to include at least one example, though maybe that doesn’t need to be replicated.